core/src/main/java/com/csun/game/UI/impl/settings/PauseSettingsTable.java

```diff
@@ -0,0 +1,98 @@
+package com.csun.game.UI.impl.settings;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.graphics.g2d.freetype.FreeTypeFontGenerator;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.ui.Image;
+import com.badlogic.gdx.scenes.scene2d.ui.ImageTextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+import com.badlogic.gdx.scenes.scene2d.utils.TextureRegionDrawable;
+import com.csun.game.attributes.Attributes;
+import com.csun.game.player.Player;
+import com.google.inject.Inject;
+
+public class PauseSettingsTable extends Table {
```

Comment:
if this class is a separate interface/window that pops up then it should extend BaseUI